docs/release_notes/version_chart/versions_support.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Prophecy versions support
+id: versions_support
+description: Prophecy versions support
+sidebar_position: 1
+tags: [compatibility, matrix, version, chart, library, plib, plibs]
+---
+
+This versions support page describes the Prophecy version system, version types, and version lifecycles.
+
+## Release version system
+
+The following table shows details of the different Prophecy version types.
+
+| Version type                      | Example       | Frequency (approx.) | End-of-support   |
+| --------------------------------- | ------------- | ------------------- | ---------------- |
+| Extended Maintenance (EM) release | `v3.4.1.0 EM` | Every four months   | After one year   |
+| Major                             | `v3.4.0.0`    | Every four months   | After six months |
+| Minor                             | `v3.3.11.0`   | Every three weeks   | After six months |
+| Patch                             | `v3.3.11.7`   | When needed         | After six months |
+
+## Extended Maintenance release
+
+Extended Maintenance (EM) releases provide you with a long-term support Prophecy version. They provide the following benefits:
```

Comment:
```suggestion
Extended Maintenance (EM) releases provide you with a long-term support Prophecy version, along with the following benefits:
```
to avoid saying provide twice